site/docs/design-docs/015-mqtt-connector.md

```diff
@@ -0,0 +1,300 @@
+---
+title: 015 - Mqtt Connector
+description: Mqtt Connector (source and sink)
+---
+
+*Target Release*: 4.3
```

Comment:
Target release can't be 4.x, because the connector is in the contrib repo. I still think it shouldn't be there. It should be in the main repo, because this should be a professional source/sink (written by us). But we will probably need to take a new look at how we add such extensions, because we are starting to have many of them and they are all in the main distribution jar.